client/my-sites/checkout/src/payment-methods/credit-card/credit-card-fields.tsx

```diff
@@ -120,6 +120,10 @@ export default function CreditCardFields( {
 			'::placeholder': {
 				color: theme.colors.placeHolderTextColor,
 			},
+			borderRadius: '2px',
```

Comment:
This is the right radius, but it's on my mind to systematize rather than hardcode. I don't think we can do that in tsx files, so there's probably no action item, but I'm throwing out a ping to some of our resident system experts, @ciampo and @mirka regardless, in case they have time to share input!

Reply:
Good call, I've added it to the theme object. That will make it easier to update in the future, although hopefully this [code is removed](https://github.com/Automattic/wp-calypso/pull/101492#issuecomment-2747251009).